pageserver/src/http/openapi_spec.yml

```diff
@@ -517,61 +517,61 @@ paths:
                 $ref: "#/components/schemas/ServiceUnavailableError"
 
   /v1/tenant/{tenant_id}/attach:
+
+  /v1/tenant/{tenant_id}/location_config:
```

Comment:
Does this mean that attach now has the same doc as location_config?

Reply:
Argh.  Good catch -- this was probably a merge snafu.  I've reinstated the attach definition.